R/execution.R

```diff
@@ -514,7 +514,28 @@ if (conceptCheckThresholdLoc == "default"){
   sql <- SqlRender::render(sql = "select * from @cdmDatabaseSchema.cdm_source;",
                            cdmDatabaseSchema = cdmDatabaseSchema)
   sql <- SqlRender::translate(sql = sql, targetDialect = connectionDetails$dbms)
-  metadata <- DatabaseConnector::querySql(connection = connection, sql = sql)
+  metadata <- tryCatch(
+    expr = {
+      DatabaseConnector::querySql(connection = connection, sql = sql)
+    },
+    error = function(e) {
+      ParallelLogger::logWarn("Retrieving metadata from cdm_source failed.")
```

Comment:
@clairblacketer We could change this into something like 
> "CDM_SOURCE not found, your DQD result will not contain metadata on your run. Are you sure you want to continue?". 

And then you have to manually 'click away' the warning.